src/styles/app/32-command-palette.css

```
@@ -31,18 +32,22 @@
 }
 
 .commandPaletteBody[data-with-preview="true"] {
-	flex-direction: row;
-	flex: 1 1 0;
+	--command-palette-preview-width: min(360px, 42%);
+	position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)